$MirOS: ports/emulators/qemu/patches/patch-target-i386_helper2_c,v 1.1 2005/11/15 12:25:09 tg Exp $
--- target-i386/helper2.c.orig	Sun Sep  4 17:11:09 2005
+++ target-i386/helper2.c	Tue Nov 15 11:37:45 2005
@@ -124,7 +124,7 @@ CPUX86State *cpu_x86_init(void)
                 env->cpuid_model[i >> 2] |= c << (8 * (i & 3));
             }
         }
-#ifdef TARGET_X86_64
+/* #ifdef TARGET_X86_64  -- You must enable it for Mac OS X x86 */
         /* currently not enabled for std i386 because not fully tested */
         env->cpuid_features |= CPUID_APIC;
         env->cpuid_ext2_features = (env->cpuid_features & 0x0183F3FF);
@@ -133,7 +133,7 @@ CPUX86State *cpu_x86_init(void)
 
         /* these features are needed for Win64 and aren't fully implemented */
         env->cpuid_features |= CPUID_MTRR | CPUID_CLFLUSH | CPUID_MCA;
-#endif
+/* #endif -- To run Mac OS X x86  */
     }
     cpu_single_env = env;
     cpu_reset(env);
